IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerResponseHandler.m

```diff
@@ -83,6 +87,16 @@ - (MSIDBrokerResponse *)brokerResponseFromEncryptedQueryParams:(NSDictionary *)e
         return nil;
     }
     
+    // In case MSAL requests AT POP but Broker response Bearer, we need to update authScheme
+    if ([authScheme isMemberOfClass:MSIDAuthenticationSchemePop.class])
+    {
+        NSString *tokenType = [decryptedResponse msidObjectForKey:MSID_OAUTH2_TOKEN_TYPE ofClass:[NSString class]];
+        if (!tokenType || MSIDAuthSchemeTypeFromString(tokenType) != MSIDAuthSchemePop)
```

Comment:
Same comment as above.

Reply:
Addressed. Will update in the next commit.